### PR TITLE
[tests-only] [full-ci] Enhance share copy-overwrite test scenarios

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
WIP - first see how these test scenarios go on ocis. Then decide what behavior is desired etc.

## Related Issue
New and changed test scenarios from core https://github.com/owncloud/core/pull/40308

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by micbar** [comment]
@phil-davis Still a thing? Or close?

File: tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature [hunk 1]
```diff
@@ -282,9 +285,72 @@ Feature: copy file
...
+    And the content of file "/Shares/sharedfile1 (2).txt" for user "Alice" should be "file to share"
+    Examples:
+      | dav_version |
+      | old         |
+      | new         |
+
+
+  Scenario Outline: copy a file over the top of an existing file received as a user share
+    Given using <dav_version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature:320]: we already have this  test:
https://github.com/owncloud/ocis/blob/10bf7f13763bf0faff6bae20fbd74fae547239c7/tests/acceptance/features/coreApiWebdavProperties/copyFile.feature#L216

File: tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature [hunk 2]
```diff
@@ -282,9 +285,72 @@ Feature: copy file
...
+    And as "Alice" file "/textfile1.txt" should exist
+    And as "Brian" file "/sharedfile1.txt" should exist
+    And the content of file "/sharedfile1.txt" for user "Brian" should be "ownCloud test text file 1"
+    Examples:
+      | dav_version |
+      | old         |
+      | new         |
+
+
+  Scenario Outline: copy a folder over the top of an existing folder received as a user share
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature:337]: we already have this test:
https://github.com/owncloud/ocis/blob/10bf7f13763bf0faff6bae20fbd74fae547239c7/tests/acceptance/features/coreApiWebdavProperties/copyFile.feature#L290

File: tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature [hunk 3]
```diff
@@ -879,3 +945,19 @@ Feature: copy file
     Examples:
       | dav_version |
       | spaces      |
+
+
+  Scenario Outline: copying a folder with a file onto another folder
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavProperties1/copyFile.feature:950]: similar test scenario:
https://github.com/owncloud/ocis/blob/10bf7f13763bf0faff6bae20fbd74fae547239c7/tests/acceptance/features/coreApiWebdavProperties/copyFile.feature#L1005

**Review by saw-jan — COMMENTED** [review]
By the looks of it, we already have these test scenarios. So, maybe we can close this one?
CC @phil-davis

**Pull request closed without merging**